adserver/templates/adserver/reports/all-geos.html

```diff
@@ -0,0 +1,15 @@
+{% extends "adserver/reports/base-all.html" %}
+{% load humanize %}
+{% load i18n %}
+
+
+{% block title %}{% trans 'All Geo Report' %}{% endblock %}
```

Comment:
Do you think it's better to name these the "Staff Geo Report" or something along those lines? This "all X" sort of breaks down for uplift.

Reply:
Yea, this is better naming.